lib/presenter/subject_add.dart

```diff
@@ -43,13 +43,14 @@ class _SubjectAddState extends State<SubjectAdd> {
   }
 
   Subject _submit() {
+    Subject result;
```

Comment:
it was intended to not return anything if the formKey.currentState.validate() is not true, so the user has the possibility to fix all wrong inputs he made.

Reply:
We need it for architecture.